.github/workflows/generate_java_sources.yml

```diff
@@ -31,4 +31,15 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: generated-sources-java
-          path: ./generated-sources/java
\ No newline at end of file
+          path: ./generated-sources/java
+      
+      - name: Send success notification
+        run: |
+          chmod +x ./scripts/send-sdk-build-notification.sh
+          ./scripts/send-sdk-build-notification.sh generate_java_sources.yml "Java build succeeded" green_check_mark ${{ secrets.SLACK_WEBHOOK }} 
```

Comment:
you can define environment variables and use them in shell scripts, maybe this is more adapted than passing a secret as a command-line argument?
https://github.community/t/using-github-action-environment-variables-in-shell-script/18330